beets/ui/commands.py

```diff
@@ -1324,8 +1324,21 @@ def stats_func(lib, opts, args):
 # version: Show current beets version.
 
 def show_version(lib, opts, args):
+    # Use the improved 'distro' package for platform info, if available
+    # (the stdlib platform module has simplistic logic, and 3.7 will drop it)
+    from platform import platform
+    os_id = platform()
+    if os.name == 'posix':
+        try:
+            import distro
+        except ImportError:
+            pass
+        else:
+            os_id = distro.name(pretty=True) or platform()
+
     print_(u'beets version %s' % beets.__version__)
     print_(u'Python version {}'.format(python_version()))
+    print_(u'Platform: {}'.format(os_id))
```

Comment:
For simplicity, maybe this can go on one line? Like “Python version X, on Y”?

Reply:
* Some people parse these things, so format changes need some thinking about
 * More useful would be "beet version @ install dir", "py x.y @ install dir", and the platform – pip does `pip 9.0.1 from ~/.local/lib/python3.4/site-packages (python 3.4)`
`